6001-7000/LOC6654CentralTegray.xml

```diff
@@ -62,7 +62,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     <note>
                         <date type="foundation" notBefore="1991">1991</date>
                         <ab type="description">The Central Tǝgrāy Zone is a subdivision of the <placeName ref="LOC5932Tegray">Tǝgrāy National Regional State</placeName> with
-                            capital <placeName ref="LOC1310Aksum">ʾAksum</placeName>.</ab>
+                            the capital <placeName ref="LOC1310Aksum">ʾAksum</placeName>.</ab>
                     </note>
                  
                 </place>
```

Comment:
here the formerlyListedAs relation is missing - it must always be inserted when a duplicate record is deleted

Reply:
?

Reply:
I did (the ID was never used).